packages/components/src/modal/stories/index.tsx

```diff
@@ -75,6 +76,8 @@ const Template: ComponentStory< typeof Modal > = ( {
 						anim id est laborum.
 					</p>
 
+					<InputControl style={ { marginBottom: '20px' } } />
+
```

Comment:
I added this so that we can check it in the storybook, is this necessary?

Reply:
It isn't _necessary_, but I think it's ok to keep around for a couple good reasons:

- It has to be tested manually (can't be put in an automated test)
- It's useful for this to be easily testable on many different platforms, since the behavior can vary between them
- It doesn't clutter the story file